Sources/Honeycomb/Honeycomb.swift

```diff
@@ -96,13 +96,17 @@ public class Honeycomb {
             )
         }
 
-        let spanExporter =
+        var spanExporter =
```

Comment:
Nit: for the sake of consistency we might want to just assign the result of an if statement to the variable similar to how we are selecting protocols.